### DISCUSSION TITLE
What does the question mark mean?

### DISCUSSION THREAD

**Discussion by AutonomousCat** [dbeaver/dbeaver#22142] [Q&A] [answered]
The tiny blue one here:
![image](https://github.com/dbeaver/dbeaver/assets/130127508/69793547-bdeb-4e9e-b3d3-fa072a639eee)

**Comment by mkgrgis** [comment, marked as answer]
This is key sign like one of 🗝 🔑 ⚿ characters.

**Reply by AutonomousCat** [reply]
wow, my eyes lmao
